integration/spark2/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParser.scala

```diff
@@ -173,6 +174,18 @@ class CarbonHelperSqlAstBuilder(conf: SQLConf,
 
     // Ensuring whether no duplicate name is used in table definition
     val colNames = cols.map(_.name)
+
+    // do not allow below key words as column name
+    colNames.foreach { col =>
```

Comment:
Please check scenario for CTAS... create table t1 as select sum(column1) as positionReference from table. In this case name of the column in t1 will be positionReference. So in that case also it should not allow to create carbon table.